### ISSUE TITLE
Program exception detected: "JsonManager.SeriesObject' does not contain a definition for 'content' in method 'AppendSearchFeed'

### ISSUE THREAD

**Issue by Gaalen69** [rrirower/Channel-Feed-Manager#93]
When I attempt to create a search feed from my series, I get a fatal exception error and the program freezes until I kill the process with Task Manager. No search feed is generated.

**To Reproduce**
1. Open Channel Manager
2. Open Channel Feed
3. Select series object
4. Select create search feed from 'Tools' dropdown menu

**Expected behavior**
I expected the program to create a search feed and save it in the search feed folder

**Screenshots**

**Screen Capture**
I have attached a screen capture of the resulting error when I attempt to create a search feed.
![Channel Feed Manager Error](https://github.com/rrirower/Channel-Feed-Manager/assets/86689420/c9c4d1ba-23b0-47ef-b7a8-d45e1ef23ebe)
![Channel Feed Manager Error](https://github.com/rrirower/Channel-Feed-Manager/assets/86689420/a0ba70ba-fcd9-4039-bd81-7f8a6ff88bfa)

**Desktop (please complete the following information):**
 - OS: Windows 10

**Additional context**
When I first got the program, I was able to create a search feed from my series object. Now that I have completed the first series and added another to the feed, I get this exception error when I try to create a search feed for one of the series objects.

**Comment by rrirower** [comment]
What version are you running?
Can you post a feed file with the series that triggers the error?

**Comment by Gaalen69** [comment]
Version 1.0.0.6

I am attaching the feed file. Thanks!
[Nation Extreme Wrestling JSON feed file.txt](https://github.com/rrirower/Channel-Feed-Manager/files/12612559/Nation.Extreme.Wrestling.JSON.feed.file.txt)

When I attempt to export the JSON feed file to Amazon MRSS feed. It just exports the xml header and none of the data. Am I doing something wrong?

**Comment by Gaalen69** [comment]
This isn't my day. I was attempting to create an Amazon MRSS file from scratch. When I discovered it only had short form video options, I went to delete it, but ended up deleting my JSON feed file. I took an old copy and put it into the feed folder, but the Channel Manager doesn't see it. Is there anything I can do?

**Comment by rrirower** [comment]
You need to add it to your library from the main screen.  If you've deleted it outside of the CFM (ie. using Windows Explorer)  You may have corrupted the channel master file.

**Comment by Gaalen69** [comment]
Thanks so much. Reinstalled and it recognized the file. Also it cleared up the exception error. However, when I export the JSON feed to MRSS, I just get the xml header and not the contents of the file. Am I doing something wrong? I've attached the xml file. Thank you for your patience. I really appreciate it.
[NEW xml export.txt](https://github.com/rrirower/Channel-Feed-Manager/files/12612846/NEW.xml.export.txt)

**Comment by rrirower** [comment]
I took a look using the json feed you provided and have found that this is a bug in the code that processes a Series while building a Search feed.  I'll have to create a fix for it.

Did you mean to export to Amazon MRSS?

**Comment by rrirower** [comment]
I ran the Export function for a large JSON feed file and it worked correctly.  I think your problem with the export is caused by the lack of Category objects within your JSON feed.  Since your JSON feed is missing Category objects, the Export code cannot "bucket" your video content into the correct categories.

**Comment by Gaalen69** [comment]
Thank you. I appreciate your help!

**Comment by Gaalen69** [comment]
You mentioned Categories not being included in my Feed. I checked the interface and when I went to use the Category field in the Series window, there was no dropdown window list to select a category from. Am I doing something wrong? Is there a place for me to input categories? Sorry if this is a silly question.
![Channel Feed Manager Blank Categories List](https://github.com/rrirower/Channel-Feed-Manager/assets/86689420/f22c60b5-4f0b-4c21-b6ec-ad067130346e)
